parser_library/src/checking/asm_instr_check.cpp

```diff
@@ -1123,7 +1125,9 @@ bool cattr::check(const std::vector<const asm_operand*>& to_check,
         else if (auto complex_op = get_complex_operand(operand); complex_op)
         {
             // operand is complex
-            const static std::vector<std::string> complex_operands = { "RMODE", "ALIGN", "FILL", "PART", "PRIORITY" };
+            const static std::vector<std::string_view> complex_operands = {
+                "RMODE", "ALIGN", "FILL", "PART", "PRIORITY"
```

Comment:
This is really handled by clang-format. While it can be forced to the string-per-line format by attaching `,` to the last argument, I actually prefer these lists to be as short as possible.